great_expectations/expectations/expectation.py

```diff
@@ -296,8 +297,15 @@ def __new__(cls, clsname, bases, attrs):
         return newclass
 
 
+class ExpectationBase(pydantic.BaseModel):
+    class Config:
+        arbitrary_types_allowed = True
+
+    pass
```

Comment:
Sorry, I should have suggested this earlier. If you don't intend this to be extended, I would make this private.
```python
class _ExpectationBase(pydantic.BaseModel)
   ...
```

Alternatively, just get rid of this class...
https://docs.pydantic.dev/1.10/usage/model_config/
```python
@public_api
class Expectation(pydantic.BaseModel, arbitrary_types_allowed = True)
   ...
```